src/main/java/org/jenkinsci/plugins/sshsteps/steps/CommandStep.java

```diff
@@ -49,7 +49,7 @@ public String getFunctionName() {
 
     @Override
     public String getDisplayName() {
-      return getPrefix() + "Execute command on remote node.";
+      return getPrefix() + "sshCommand - Execute command on remote node.";
```

Comment:
Can this use `getFunctionName`?

Reply:
Nice catch! Updated with https://github.com/jenkinsci/ssh-steps-plugin/pull/2/commits/a89f5da1a19bd2473493836356fae3d02e795c1f